solutions/56.js

```diff
@@ -27,7 +27,24 @@ const solution = (arr, total) => {
     return acc;
   }, total);
 };
+// return min number of coins
+// daniel s.
+const solution2 = (arr, value) => {
+  if (value === 0) {
+    return 0;
+  };
+  let currentMin = value;
+  arr.forEach((coin)=> {
+  let childCoins = 0;
+    if (value - coin > 0) {
+      childCoins = solution2(arr, value - coin);
+    }
+    currentMin = Math.min(currentMin, childCoins);
```

Comment:
Just one comment for more concise code in the future: If `value - coin` is not greater than 0, than we do not need to `currentMin`. 

Other than that, the changes are approved.